### PR TITLE
Adding brand assets

### PR DESCRIPTION
**Pull request by panchovm**
## Description

Adding other brand assets for media uses in the `brand` folder.

## Assets added
These assets come from the following Design Library' sections.

imagen

CleanShot 2023-03-10 at 10 33 37@2x

### CODE REVIEW

**Review by obulat — COMMENTED** [review]
Are these assets for use on the frontend or in documentation? If it is for the frontend, we should move it to `frontend/src/assets`.

**Comment by panchovm** [comment]
Documentation. I saw a few people using old or wrong assets for social media or other non-frontend graphics.

**Review by obulat — APPROVED** [review]
Thank you for adding them!

**Review by dhruvkb — CHANGES_REQUESTED** [review]
@panchovm could you update the `README.md` file in the `brand/` directory with some guidelines on when to use which of these SVGs? To someone without that context, these SVGs seem pretty much the same and very easy to use the wrong one for the job.

**Comment by panchovm** [comment]
I planned to do that, but PR seemed to work just for uploading the files. I might be wrong. If so, I definitely want to edit the `README.md` file.

File: brand/README.md [hunk 1]
```diff
@@ -35,16 +35,38 @@ color them. The SVG paths for the icon are classed based on their position as
...
+| --- | --- |
+| Transparent | ![Thumbnail small Transparent](thumbnail-small-transparent.svg) |
+| Yellow | ![Thumbnail small Yellow](thumbnail-small-yellow.svg) |
+| Dark | ![Thumbnail small Dark](thumbnail-small-dark.svg) |
+
+### Icon
+
+Icon size as [WordPress Design Library](https://www.figma.com/file/e4tLacmlPuZV47l7901FEs/WordPress-Design-Library?node-id=8%3A13) guidelines.
+
+![Openverse icon](wp-icon-openverse.svg)
```
Comment (obulat) [inline brand/README.md:72]:
```suggestion
![Openverse icon](wp-icon-openverse.svg)

```

**Review by dhruvkb — APPROVED** [review]
I still have no idea when to use which thumbnail 😅 but we can work on that separately. No need to block this PR over such a minor detail.

**Comment by zackkrida** [comment]
@dhruvkb I've made a new issue for the discussion of branding guidelines: https://github.com/WordPress/openverse/issues/901

**Pull request merged**